extensions/machine-learning/src/common/queryRunner.ts

```diff
@@ -19,8 +19,7 @@ EXEC sp_execute_external_script
 @script=N'import pkg_resources
 import pandas
 OutputDataSet = pandas.DataFrame([(d.project_name, d.version) for d in pkg_resources.working_set])'
-select e.name, version from sys.external_libraries e join @tablevar t on e.name = t.name
-where [language] = 'PYTHON'
+select t.name, (CASE   WHEN e.name is NULL THEN 1 ELSE 0 END) as isReadOnly , version from @tablevar t left join sys.external_libraries e  on e.name = t.name and e.[language] = 'PYTHON'
```

Comment:
spacing in this and the other query looks weird, but I have no idea how pandas is supposed to look!